### ISSUE TITLE
[Meta] Enable working with the API aggregator

### ISSUE THREAD

**Issue by pmorie** [kubernetes-incubator/service-catalog#458]
This issue is to enable operating the catalog behind the k8s API aggregator.  Depends on #457.

Details to figure out:

- [ ] Delegated authn/authz back to the core API server (https://github.com/kubernetes-incubator/service-catalog/issues/457)
- [ ] SSL from the aggregator to the API server (https://github.com/kubernetes-incubator/service-catalog/issues/389)

**Comment by arschles** [comment]
Ref https://github.com/kubernetes-incubator/service-catalog/issues/389, since we may have to figure out how to enable the aggregator to talk to our API server via SSL

**Comment by DirectXMan12** [comment]
see #615

**Comment by DirectXMan12** [comment]
(also, feel free to assign this to me)

**Comment by arschles** [comment]
Moved to milestone `1.0.0`, as that's when we plan to have full support for the aggregator. Please see [the roadmap](https://github.com/kubernetes-incubator/service-catalog/wiki/Roadmap#100)

**Comment by pwittrock** [comment]
Ack

**Comment by duglin** [comment]
@MHBauer can this be closed now?

**Comment by pmorie** [comment]
I believe that this can be closed now, we have this working with the 1.7 aggregator.  If there are further issues that need to be solved, someone should open a new issue.
